### ISSUE TITLE
Permission to use this (strapi-gatsby-porfolio-2020-api) tutorial 

### ISSUE THREAD

**Issue by adriatic** [john-smilga/strapi-gatsby-porfolio-2020-api#6]
Hello, John

I have yet to find better-composed tutorials than yours, so it would be a great service to my open source project to be built in the Github organization **https://github.com/iam-concepts/**. The project is dedicated to Strapi and will be built by the Strapi community after I take a good whack at it first. The common theme is IAM (Identity and access management) so, the layout will be like this 

1. Start with the strapi-gatsby-porfolio-2020-api which will live in the **[basics](shttps://github.com/iam-concepts/basics)** repository, as the Step 1 in my planned set of full-stack apps, all using Strapi as the back-end

2. The [Documentation](https://github.com/iam-concepts/documentation

3. The front-end family of apps will include all framework of interest to the Strapi community

At this point, I start building and describing the AIM functionality for all full-stack apps already built. Just like the case with front-end apps, the AIM functionality will cover many different IP (Identity Providers) of interest to the Strapi community.

All documentation will be written using Vuepress or **[MDsveX](https://github.com/pngwn/)**

While your sample will be just a small part (an introduction) into **[IAM Concepts](ttps://github.com/iam-concepts/)** projects, it is critically important to start with something well designed. You will be fully credited as the author - and I even dare to think that you may join me in some fashion. 

What say you?

Regards

Nik

**Comment by john-smilga** [comment]
Hello, Nik

Please don’t take it personally but I have a hard time understanding what you are going to build. With that said you can use the repo with one condition - I’m not responsible for any updates or issues. In fact I can tell you right away that I will ignore all the open issues and pull requests. 
In my experience Strapi is CONSTANTLY coming up with new (breaking) changes/updates (plugins and the CMS itself) which is frankly extremely annoying. The whole point of Strapi was to get up and running with API in matter of minutes not to spend weekends fiddling around with my backend. Because of that I stopped using Strapi and don’t suggest it to my students as well. Even this repo is technically out of date, since one of the plugins got an update (again).

Best,
John

**Comment by adriatic** [comment]
> Please don’t take it personally but I have a hard time understanding what you are going to build.

Hi, John

No wonder you had a hard time understanding what am I  planning to build - I have an equally hard time based only on rereading my message. I must have been very confused writing it.

My intent is to teach the basics of **Identity and Access Management** on a set of simple full-stack applications. Since the whole effort is meant to help Strapi's popularity, this family of samples will have one constant - Strapi backend, while front-ends  and Identity Providers will vary.

The idea to use your app as a starting point seems less than brilliant since I sense your dissatisfaction with Strapi - and I cannot simply ignore it. In addition, I changed my mind because I realized that the Strapi's own **[Quick Start](https://strapi.io/documentation/v3.x/getting-started/quick-start.html)** is a better choice in this case where I want minimal front-end / back end pair in order to IAM stuff gets the reader's attention.

I never assumed that you would be supporting this code - but that is also not a good assumption; if I just fork and support your app myself I would create a monstrosity 😄 

In summary, let's just perceive my message as a sign of appreciation for your **[strapi-gatsby-porfolio-2020](https://github.com/john-smilga/strapi-gatsby-porfolio-2020-api)**.

Best regards

Nik

**Comment by john-smilga** [comment]
Hey Nick,

Got it.Sorry if I sounded too negative, just frustrated with Strapi, please don’t take it personally.

**Comment by adriatic** [comment]
With your permission and the desire to help Strapi, I will share your feedback with Strapi team (anonymously, of course)

**Comment by john-smilga** [comment]
Sure.
